bgpd/bgp_attr.h

```diff
@@ -287,6 +288,12 @@ struct attr {
 
 	/* EVPN ES */
 	esi_t esi;
+
+	/* SR-TE Policy Name */
+	char srte_policy[SRTE_POLICY_NAME_MAX_LENGTH];
```

Comment:
Is this actually an attribute? Attributes are for data on the wire.  Yes I realize that this has leaked in but let's start fixing this

Reply:
This is in fact not used anywhere, probably some remnent of a first implementation. Good catch.